cmd/nodecmd/create.go

```diff
@@ -339,34 +382,40 @@ func GetLatestAvagoVersionForRPC(configuredRPCVersion int) (string, error) {
 // promptAvalancheGoReferenceChoice returns user's choice of either using the latest Avalanche Go
 // version or using the latest Avalanche Go version that is still compatible with the subnet that user
 // wants the cloud server to track
-func promptAvalancheGoReferenceChoice() (string, error) {
+func promptAvalancheGoReferenceChoice() (string, string, error) {
 	defaultVersion := "Use latest Avalanche Go Version"
 	txt := "What version of Avalanche Go would you like to install in the node?"
 	versionOptions := []string{defaultVersion, "Use the deployed Subnet's VM version that the node will be validating"}
 	versionOption, err := app.Prompt.CaptureList(txt, versionOptions)
 	if err != nil {
-		return "", err
+		return "", "", err
 	}
 
 	switch versionOption {
 	case defaultVersion:
-		return "latest", nil
+		return "latest", "", nil
```

Comment:
"latest" and "subnet" convert to constants

Reply:
ok